source/qml/SettingsPage.qml

```diff
@@ -91,6 +91,8 @@ ScrollView {
         settingsHolder.msatidx = satelliteSelectorCombo.currentIndex
 
         settingsHolder.uu = uploadUsernameTF.text.trim();
+        settingsHolder.up = uploadPasswordTF.text;
```

Comment:
User password saving was left out on purpose. Persisting the user's password as a cleartext value in a potentially easily accessible place (e.g. Windows registry, etc.) is not a good idea (in my opinion).